packages/editor-ui/src/stores/workflows.store.ts

```diff
@@ -694,7 +694,7 @@ export const useWorkflowsStore = defineStore(STORES.WORKFLOWS, {
 			}
 
 			const storedPinData = payload.data.map((item) =>
-				isJsonKeyObject(item) ? item : { json: item },
+				isJsonKeyObject(item) ? { json: item.json } : { json: item },
```

Comment:
does this also remove `binary` from `item`?

Reply:
We don't support pinning binary data at the moment. 🤷🏻‍♂️